### ISSUE TITLE
Scrolling in tab bar

### ISSUE THREAD

**Issue by plinss** [sublimehq/sublime_merge#762]
**Problem description**

I opened more repository tabs than fit within the window, collapsing the display of a number of tabs on the left.

It was somewhat difficult to select the leftmost tab. Some horizontal scrolling controls like SublimeText has for file tabs would have come in handy.

Note that I did, later, find that using the vertical scroll wheel on my mouse did scroll the tabs. But using a vertical scroll to scroll horizontally isn't intuitive (and that mouse only scrolls vertically).

**Preferred solution**

Some visble scrolling control for repo tabs. Preferably matching the paradigm used in SublimeText.

**Alternatives**

A popup control giving a menu of all open repos within the window would also be nice.

**Comment by dpjohnst** [comment]
Hi @plinss,

Thank you for sharing your feedback with us!

A tab list popup (similar to Sublime Text) is definitely on our radar.

With regards to the scrolling, we should match the same tab scrolling behavior as Sublime Text.
i.e. both horizontal and vertical scrolling should scroll the tab bar.
Let me know if this isn't the case for you, and I can investigate further :)

Kind regards,
\- Dylan

**Comment by plinss** [comment]
Yes, vertical scrolling does in fact scroll the tab bar horizontally. I just wasn't expecting that to work initially and found myself clicking through tabs trying to get the one I wanted to be visible (such is life on Windows with a cheap mouse, on my Mac I can scroll horizontally easily). A pair of scroll buttons like SublimeText has would've given me an obvious solution, being used to those I was looking for them.

Supporting the vertical scroll wheel is a good compromise fallback, it's just not obvious to someone used to having horizontal scrolling on their mouse.

**Comment by dpjohnst** [comment]
Hi @plinss,

I have tweaked the title slightly, and I've changed this to an enhancement rather than a bug.
I'll continue to keep you updated of any progress on this request.

Another related issue you might be interested in upvoting: https://github.com/sublimehq/sublime_merge/issues/723

Thanks,
\- Dylan

**Comment by plinss** [comment]
All sounds good, thanks!
